samples/sqltransport/native-timeout-migration/SqlServer_3.1/Endpoint.Native/MigrateFromNHibernate.sql

```diff
@@ -0,0 +1,24 @@
+﻿-- startcode MigrateFromNHibernate
+
+declare @endpointName nvarchar(max) = 'PUT ENDPOINT NAME HERE';
+
+declare @endpointSchema nvarchar(max) = 'dbo';
+declare @dalayedTableSuffix nvarchar(max) = 'Delayed';
+
+declare @migrateScript nvarchar(max);
+set @migrateScript = '
+    WITH message AS (
+		SELECT * 
+		FROM TimeoutEntity WITH (UPDLOCK, READPAST, ROWLOCK) 
+		WHERE [Endpoint] = ''' + @endpointName + '''
+			AND [Time] IS NOT NULL)
+	DELETE FROM message
```

Comment:
Smells like unbounded delete. Maybe we should add a note that if number of scheduled timeouts is significant, then add batching.

Or maybe turn this into batched deletes right from the start?